src/models/OtherReason.schema.ts

```diff
@@ -0,0 +1,25 @@
+import * as Joi from '@hapi/joi';
+
+const titleErrorMessage = 'You must give your reason a title';
+const descriptionErrorMessage = 'You must give us more information';
+
+export const schema = Joi.object({
+    title: Joi.string()
+        .required()
+        .pattern(/\w+/)
```

Comment:
I dont think that regex is necessary, as in the message 'string.empty' already captures cases where nothing is input